### PR TITLE
[16.0][MIG] stock_product_pack: Migration to 16.0

### PR DESCRIPTION
**Pull request by rven**

### CODE REVIEW

**Review by FrancoMaxime — COMMENTED** [review]
There are 6 commits with the same label "[MIG] stock_product_pack: Migration to 15.0", but on the branch 15.0 there is only one. 
Is it possible to squash them? (its a question, not a requirement)

**Comment by rven** [comment]
> There are 6 commits with the same label "[MIG] stock_product_pack: Migration to 15.0", but on the branch 15.0 there is only one. Is it possible to squash them? (its a question, not a requirement)

It was because I started the conversion of an other v15.0 PR, I restarted the conversion from the migrated version now.

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo-addon-product-pack @ git+https://github.com/OCA/product-pack.git@refs/pull/120/head#subdirectory=setup/product_pack
```
Comment (FrancoMaxime) [inline test-requirements.txt]: I think It is not needed anymore since product_pack has been merged

**Review by FrancoMaxime — APPROVED** [review]

File: stock_product_pack/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# Copyright 2020 Tecnativa - João Marques
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock product Pack",
+    "version": "16.0.1.0.0",
+    "category": "Warehouse",
+    "summary": "This module allows you to get the right available quantities "
+    "of the packs",
+    "website": "https://github.com/OCA/product-pack",
+    "author": "NaN·tic, " "ADHOC SA, " "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (FrancoMaxime) [inline stock_product_pack/__manifest__.py]:
```suggestion
    "author": "NaN·tic, ADHOC SA, Tecnativa, Odoo Community Association (OCA)",
```

**Review by FrancoMaxime — APPROVED** [review]
LGTM, a minor remark with an old pre_commit error

**Review by FrancoMaxime — APPROVED** [review]
LGTM, code review

**Review by alvaro-domatix — APPROVED** [review]

**Pull request merged**